### ISSUE TITLE
Enhancement request: Multi-file selection.

### ISSUE THREAD

**Issue by Mechscientist** [CauldronDevelopmentLLC/CAMotics#307]
Hi,

Can you modify the file selector for adding g-code files to a project to allow multi-selection? The fact this it's single select and doesn't remember the last directory/folder is somewhat annoying.

Thanks!

**Comment by jcoffland** [comment]
Yeah, not remembering the last folder is something I've tried to fix but still haven't sorted out.

Note, that adding multiple GCode files causes CAMotics to processes them in order, one after another.
